config/helm/chart/default/Chart.yaml

```diff
@@ -18,8 +18,8 @@ description: The Dynatrace Operator Helm chart for Kubernetes and OpenShift
 icon: https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_512x512px.png
 home: https://www.dynatrace.com/
 type: application
-version: 0.4.2
-appVersion: 0.4.2
+version: 0.6.0
```

Comment:
shouldn't this be `snapshot` for master?

Reply:
snapshot does not work, has to be semver

Reply:
😢